iree/compiler/Codegen/LLVMGPU/LLVMGPUUtils.h

```diff
@@ -13,6 +13,8 @@
 
 static constexpr int32_t kNumGPUDims = 3;
 
+static constexpr int32_t kWarpSize = 32;
```

Comment:
I just noticed that these declarations are not put in a namespace. Could you put them in the proper namespace to avoid leaking to the global namespace?

Reply:
Those are marked as static so they don't pollute  anything outside of the files including this header similar to the static functions.
Adding mlir and iree namespace for the whole file.